docs/community/contributing.md

```diff
@@ -1,6 +1,8 @@
 % NOTE(vytas): This document is in MyST markdown (unlike others) in order to be
 %   able to share CONTRIBUTING.md verbatim.
 
+(contribute)=
```

Comment:
is this an anchor? sorry but I'm not familiar with this md syntax.

Reply:
Me neither, had to google it up. GitHub doesn't understand it either, it is a MyST extension.

Basically it is an equivalent of the following Rst:
```rst
.. _contribute:
```